gammapy/estimators/tests/test_flux.py

```diff
@@ -84,54 +91,43 @@ def test_flux_estimator_fermi_with_reoptimization(fermi_datasets):
 def test_flux_estimator_1d(hess_datasets):
     estimator = FluxEstimator(
         source="Crab",
-        energy_min=1 * u.TeV,
-        energy_max=10 * u.TeV,
         selection_optional=["errn-errp", "ul"],
+        reoptimize=False
+    )
+    datasets = hess_datasets.slice_by_energy(
+        energy_min=1 * u.TeV, energy_max=10 * u.TeV,
     )
-    result = estimator.run(hess_datasets)
+    datasets.models = hess_datasets.models
+
+    result = estimator.run(datasets)
 
     assert_allclose(result["norm"], 1.218139, atol=1e-3)
     assert_allclose(result["ts"], 527.492959, atol=1e-3)
     assert_allclose(result["norm_err"], 0.095496, atol=1e-3)
     assert_allclose(result["norm_errn"], 0.093204, atol=1e-3)
     assert_allclose(result["norm_errp"], 0.097818, atol=1e-3)
-    assert_allclose(result["norm_ul"], 1.525773, atol=1e-3)
+    assert_allclose(result["norm_ul"], 1.418475, atol=1e-3)
```

Comment:
Any idea for the change here?

Reply:
Yes, the default for `n_sigma_ul` changed from 3 to 2. I guess for consistency we should always chose 2 and the change is OK?